apps/greencheck/views.py

```diff
@@ -235,21 +235,23 @@ class Meta:
         fields = ["services", "country"]
 
 
-class DirectoryView(WaffleFlagMixin, TemplateView):
+class DirectoryView(TemplateView):
     """
     A view for filtering our list of providers by various criteria
     """
 
     template_name = "greencheck/directory_index.html"
-    waffle_flag = "directory_listing"
 
     def get_context_data(self, *args, **kwargs):
         """
         Populate the page context with the filtered list
         of providers
         """
 
-        queryset = Hostingprovider.objects.filter(showonwebsite=True).prefetch_related(
+        queryset = Hostingprovider.objects.filter(
+            showonwebsite=True,
+            archived=False
+        ).prefetch_related(
```

Comment:
@mrchrisadams The only change I made was here. But this PR is showing a bunch of other files being changed too. Is that because I'm merging it into Staging rather than Main?

Reply:
hi @fershad  - I think it might be because what's on staging might not be the same as main / master.

I've typically treated staging as something we're ok to `git push --force` rather than open up PRs to `staging`. 

My reasoning is that while we care about a specific branch being merged into `master`, there are sometimes cases where we might want to see how something looks in `staging`, without then proceeding directly into  merge it into `master`, and we also make no guarantees about `staging` always working. 


```
git push origin ca-card-350-fix-ipv6-support:staging -f                                                                                                                                           
```

and in this case we'd do something like

1. update this PR to be a PR against `master`

2. force push the change to staging with something like the command below

```
git push origin improve-directory-filter:staging -f                                                                                              
```

I'll try to respond in Zulip today if you DM me, otherwise we have the call on Monday to chat as well.

Reply:
@fershad  are you ok with updating the PR so it's a change against master, a bit like the way you have a comparison view here?

https://github.com/thegreenwebfoundation/admin-portal/compare/master...add-disclaimer-for-dataset